src/SlateAsInputEditor/index.js

```diff
@@ -330,6 +330,44 @@ const SlateAsInputEditor = React.forwardRef((props, ref) => {
     return next();
   };
 
+  /**
+  * Method to handle lists
+  * @param {*} editor
+  * @param {*} type
+  */  
+ 
+  let handleList = (editor, type) => {
+    if (action.isSelectionList(editor.value)) {
+      if (action.currentList(editor.value).type === type) {
+        return action.transformListToParagraph(editor, type);
+      } else {
+        return action.transformListSwap(editor, type, editor.value);
+      }
+    } else if( action.isSelectionInput(editor.value, "block_quote") ) {
+      editor.unwrapBlock("block_quote");
+      return action.transformParagraphToList(editor, type);
+
+    }else{
+      return action.transformParagraphToList(editor, type);
+    }
+  };
+  
+    /**
+  * Method to handle block quotes
+  * @param {*} editor
+  */  
+ 
+  let handleBlockQuotes = (editor) => {
+    if(action.isSelectionInput(editor.value, "block_quote")){
+    editor.unwrapBlock("block_quote");
+    }else if(action.isSelectionList(editor.value)){
+      action.isSelectionInput(editor.value, "ol_list")?action.transformListToParagraph(editor,'ol_list'):action.transformListToParagraph(editor,'ul_list')
```

Comment:
This is a minor note for future reference, but this may be more readable: 

```js
action.isSelectionInput(editor.value, "ol_list")
  ? action.transformListToParagraph(editor,'ol_list')
  : action.transformListToParagraph(editor,'ul_list')
```

Reply:
Noted! will take care in the future.